src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task {
+    private String dateTime;
```

Comment:
What is the purpose of this dateTime as I do not see it being used anywhere within this class?